semgrep-core/tests/dockerfile/no-trailing-newline.dockerfile

```diff
@@ -0,0 +1 @@
+CMD hello
```

Comment:
hmm you miss a #ERROR: match here I think

Reply:
thanks for the fix!